code/modules/jobs/job_types/bartender.dm

```diff
@@ -36,3 +36,11 @@
 	backpack_contents = list(/obj/item/storage/box/beanbag=1)
 	shoes = /obj/item/clothing/shoes/laceup
 
+
+/datum/outfit/job/bartender/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	..()
+
+	if(visualsOnly)
+		return
+
+	ADD_TRAIT(H, TRAIT_SOMMELIER, ROUNDSTART_TRAIT)
```

Comment:
should this not apply on job change?
there should be a signal for job assignment, if not, make one

Reply:
Tracking your job through id card is the only way for it and I don't think it's a good design, but also I want this as an inherent trait to the original bartender crews since they come from NT with bartending training.

Reply:
sounds fair, but can't someone just put on the bartender outfit to do this, or is it only roundstart?

Reply:
only original bartenders. crews with beer goggles will only say some flat explanation instead of with an explanation for booze power.

Reply:
alright, looks good then